addons/mail/static/src/views/fields/emojis_text_field/emojis_text_field.js

```diff
@@ -3,24 +3,51 @@
 import { TextField } from "@web/views/fields/text/text_field";
 import { patch } from "@web/core/utils/patch";
 import MailEmojisMixin from "@mail/js/emojis_mixin";
-import { EmojisDropdown } from "@mail/js/emojis_dropdown";
 import { EmojisFieldCommon } from "@mail/views/fields/emojis_field_common";
 import { registry } from "@web/core/registry";
 
+const { onMounted, useRef } = owl;
+
 /**
  * Extension of the FieldText that will add emojis support
  */
 export class EmojisTextField extends TextField {
     setup() {
         super.setup();
         this.targetEditElement = this.textareaRef;
+        this.triggerButton = useRef("emojisButton");
+        onMounted(() => this.setupMessaging(this.triggerButton, this.targetEditElement));
         this._setupOverride();
     }
+
+    async setupMessaging(buttonRef, inputRef) {
+        const messaging = await this.env.services.messaging.get();
+        this.emojiTextField = await messaging.models["EmojiTextFieldView"].insert({
+            buttonEmojisRef: buttonRef,
+            textareaRef: inputRef,
+            onInputModifiedCallback: this.onInputModified.bind(this),
+        });
```

Comment:
Don't forget to `onWillUnmount(() => this.emojiTextField && this.emojiTextField.delete())`.
It's not a super big deal, but if you wanna make sure it's properly cleaned.